modules/install-ibm-cloud-configuring-the-public-subnet.adoc

```diff
@@ -0,0 +1,140 @@
+// This is included in the following assemblies:
+//
+// installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="configuring-the-public-subnet_{context}"]
+= Configuring the public subnet
+
+All of the {product-title} cluster nodes must be on the public subnet. IBM Cloud&#174; does not provide a DHCP server on the subnet. Set it up separately on the provisioner node.
+
+.Procedure
+
+. Install `dnsmasq`:
++
+[source,terminal]
+----
+$ sudo dnf install dnsmasq
+----
+
+. Set an unused IP address from the public IP range:
++
+[source,terminal]
+----
+$ UNUSEDIP=<IP-address>
+----
+
+. Open the `dnsmasq` configuration file:
++
+[source,terminal]
+----
+$ sudo vi /etc/dnsmasq.conf
+----
+
+. Add the following configuration to the `dnsmasq` configuration file:
++
+[source,configuration files]
+----
+interface=baremetal
+except-interface=lo
+bind-dynamic
+log-dhcp
+
+dhcp-range=$UNUSEDIP,$UNUSEDIP,$PUBLICCIDR
```

Comment:
we cannot use the variables here, we should use their values instead like
dhcp-range=141.125.65.220,141.125.65.220,28
dhcp-option=baremetal,121,0.0.0.0/0,141.125.65.209,10.196.130.131/26,141.125.65.212